docs/WindowsSubsystemForLinux.md

```diff
@@ -1,60 +0,0 @@
-# Swift on the Windows Subsystem for Linux (WSL)
-
-- Note that all compiled Swift binaries are only executable within Bash on
-  Windows and are Ubuntu, not Windows, executables (WSL can natively run Ubuntu
-  executables).
-- Make sure to run all commands from `bash`, or the project won't compile.
-
-###  1. Install WSL
-Install and run the latest version of [Bash on Ubuntu on
-Windows](https://docs.microsoft.com/en-us/windows/wsl/about) installed on
-your PC.
-```
-bash
-```
-
-### 2. Install dependencies
-Install the developer dependencies needed to compile the Swift project. These
-are identical to the Ubuntu dependencies, with the addition of `make`.
```

Comment:
Here

Reply:
Ah.  I don't see why `make` is needed.  This is just wrong.

Reply:
Maybe to build `ninja` at the very start?